doc/about/faqs.rst

```diff
@@ -0,0 +1,27 @@
+FAQS (Frequently Asked Questions)
+=================================
+
+This section contains a list of frequently asked questions.
+
+.. rubric:: How should I cite Pastas in general and/or a specific version?
+
+If you find Pastas useful and use it in your research or project, we kindly
+ask you to cite the Pastas article published in Groundwater journal as follows:
+
+- Collenteur, R.A., Bakker, M., Caljé, R., Klop, S.A., Schaars, F. (2019)
+  `Pastas: open source software for the analysis of groundwater time series.
+  Groundwater <https://ngwa.onlinelibrary.wiley.com/doi/full/10.1111/gwat
+  .12925>`_. doi: 10.1111/gwat.12925.
+
+To cite a specific version of the Pastas software we recommend using the the
+DOI proivded through Zenodo for every major release of the software.
+
+- Collenteur, R., Bakker, M., Caljé, R. & Schaars, F. (XXXX). Pastas:
+  open-source software for time series analysis in hydrology (Version X.X.X).
+  Zenodo. http://doi.org/10.5281/zenodo.1465866.
+
+.. rubric:: What does "Pastas" stand for?
+
+Pastas is not an acronym for something. However we like a good Pasta at it's
```

Comment:
tiny dutchism I think: "at it's time"
Maybe: "we like a good Pasta every once in a while"?